packages/jobs/src/workers/worker-definitions/exampleWorker.ts

```diff
@@ -4,6 +4,7 @@ import { Job, Processor } from 'bullmq'
 
 type ExampleResult = { job: Job; patata: string }
 const processor: Processor<ExampleJobData, ExampleResult> = async (job) => {
+  console.log('JOB received', job.id, job.data)
```

Comment:
Leave for now. The first person implementing a real job will review this example worker